lib/spack/spack/test/cmd/install.py

```diff
@@ -276,7 +276,7 @@ def test_install_commit(mock_git_version_info, install_mockery, mock_packages, m
     assert filename in installed
     with open(spec.prefix.bin.join(filename), "r") as f:
         content = f.read().strip()
-    assert content == "[]"  # contents are weird for another test
+    assert content == "[0]"  # contents are weird for another test
```

Comment:
Ok, this comment already existed, but it's not particularly helpful for understanding what is tested, and why it's changed now.